src/background/config_rc.ts

```diff
@@ -59,17 +59,21 @@ export async function runRc(rc: string) {
 }
 
 export function rcFileToExCmds(rcText: string): string[] {
-    // string-join lines that end with /
-    const joined = rcText.replace(/\\\n/g, "")
+   // Remove empty and comment lines
+   const rm= Array.from(rcText).filter(
```

Comment:
I used to convert the string to an Array because filter was showing error. @bovine3dom

Reply:
Have you tried running this code step by step in a REPL and seeing what it does?

The filter is expecting an array that contains one line of text per element.

`Array.from(string)` makes an array where each character in the string becomes an individual element, which isn't what we want.